### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by devdesco-ceo**
Widget embedding in Magento 2

### CODE REVIEW

File: 5. Administrator Guides/Livechat/README.md [hunk 1]
```diff
@@ -141,3 +141,25 @@ Here is an example of the JSON data sent on a livechat offline message:
...
+                w.RocketChat = function(c) { w.RocketChat._.push(c) }; w.RocketChat._ = []; w.RocketChat.url = u;
+                var h = d.getElementsByTagName(s)[0], j = d.createElement(s);
+                j.async = true; j.src = 'https://rc.example.com/packages/rocketchat_livechat/assets/rocket-livechat.js';
+                h.parentNode.insertBefore(j, h);
+            })(window, document, 'script', 'https://rc.example.com/livechat');
+        });
+    </script>
+
+```
+Replace the `https://rc.example.com/` with your Rocket.Chat (sub-)domain, and pase the code into your theme file `/app/design/frontend/Theme_provider/theme_name/Magento_Theme/templates/root.phtml` righr before the closing `</html>` tag.
```
Comment (graywolf336) [inline 5. Administrator Guides/Livechat/README.md]: * `pase` should be `paste`?
* `righr` should be `right`?

Comment (devdesco-ceo) [inline 5. Administrator Guides/Livechat/README.md]: Sorry....

**Comment by geekgonecrazy** [comment]
Only concern I have with this... is in the future if we change any of the methods for livechat we'll have to update the chunk of code here.  But might not be that big of risk :)

**Comment by devdesco-ceo** [comment]
At least for now that works.... and many people have issues on figuring out how to do JS snippets with RequireJS. LOL

**Pull request merged**